lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -917,6 +915,62 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     }
     return hasPrintedAnything;
   }
+    
+  void printRenameForDecl(const AvailableAttr *AvAttr, const ValueDecl *D,
+                          bool includeQuotes) {
+    if (AvAttr->Rename.empty())
+      return;
```

Comment:
I think you can assert this rather than early-exiting. Any caller would already have to deal with the no-rename case differently.

(Sorry I didn't make this comment last time. I guess I was distracted by the formatting.)

Reply:
OK, I'll change this to assert instead